internal/ingress/annotations/proxyssl/main.go

```diff
@@ -190,27 +191,27 @@ func (p proxySSL) Parse(ing *networking.Ingress) (interface{}, error) {
 	config := &Config{}
 
 	proxysslsecret, err := parser.GetStringAnnotation(proxySSLSecretAnnotation, ing, p.annotationConfig.Annotations)
-	if err != nil {
+	if err != nil && !errors.Is(err, ing_errors.ErrMissingAnnotations) {
 		return &Config{}, err
-	}
-
-	ns, _, err := k8s.ParseNameNS(proxysslsecret)
-	if err != nil {
-		return &Config{}, ing_errors.NewLocationDenied(err.Error())
-	}
+	} else if err == nil {
```

Comment:
why wrap it in else if? line 194 is enough for the bug fix?

Reply:
I wanted to make sure that the annotation was present. My understanding is that `err == ing_errors.ErrMissingAnnotations` will mean annotation is missing. So, I check for `err == nil` to confirm that annotation is present and there is no parsing error.